tracim/tracim/tests/functional/test_calendar.py

```diff
@@ -0,0 +1,220 @@
+import time
+
+import caldav
+import transaction
+from caldav.lib.error import AuthorizationError
+from nose.tools import eq_, ok_
+import requests
+from requests.exceptions import ConnectionError
+from sqlalchemy.orm.exc import NoResultFound
+
+from tracim.config.app_cfg import daemons
+from tracim.lib.workspace import WorkspaceApi
+from tracim.model import DBSession
+from tracim.tests import TestCalendar as BaseTestCalendar
+from tracim.model.auth import User
+from tracim.model.data import Content
+
+
+class TestCalendar(BaseTestCalendar):
+    def setUp(self):
+        super().setUp()
+        time.sleep(3)  # TODO - 20160606 - Bastien: sleep to wait ...
```

Comment:
Yes, i add a TODO, we should use a lock but it's a bit complicated to do it quickly :s